functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/WizardAction.java

```diff
@@ -158,6 +159,8 @@ public void openNewBtnLiferayComponentClassWizard() {
 	public void openNewFragmentWizard() {
 		assertTitleStartBy(_getWizard(), ide.getShell());
 
+		switchLiferayWorkspacePerspective();
```

Comment:
Have you change something on fragment? why this changed during udpating server deploy tests?